modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * @file
+ * The iNope Policy Review Module.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function inope_policy_review_menu() {
+  $items = array();
+
+  $items['node/%node/review'] = array(
+    'title callback' => 'inope_policy_review_tab_title_callback',
+    'title arguments' => array(1),
+    'description' => 'Review of policy nodes.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('inope_policy_review_form', 1),
+    'access arguments' => array('view inope policy review'),
+    'type' => MENU_LOCAL_TASK,
+  );
+  return $items;
+
+}
+
+/**
+ * Creates the iNope policy review page title.
+ *
+ * The page title will be 'Review of @node_title' where @node_title is replaced
+ * by the title of the object returned from node_load().
+ *
+ * @param $nid
+ *   String of numbers representing the node id passed from hook_menu.
+ * @return
+ *   String of characters to be used as the title in a menu local task for
+ *   policy nodes.
+ */
+function inope_policy_review_tab_title_callback($node) {
+  $node_title = $node->title;
+  $title = t('Review of @node_title', array('@node_title' => $node_title));
+  return $title;
+}
+
+
+/**
+ * Implements hook_permission().
+ */
+function inope_policy_review_permission() {
+  return array(
+    'view inope policy review' => array(
+      'title' => t('View iNope Policy Review'),
+      'description' => t('View the review tab on policy node pages'),
+    ),
+  );
+}
+
+/**
+ * The iNope Policy Review Form.
+ */
+function inope_policy_review_form($form, &$form_state) {
+  global $user;
+
+  $status = array(
+    0 => t('I have reviewed this policy and approved it for use'),
+    1 => t('I have reviewed this policy and would like to see changes'),
+  );
+
+  $form['inope_policy_review_status_radio'] = array(
+    '#type' => 'radios',
+    '#options' => $status,
+  );
+
+  $form['inope_policy_review_changes_requested_textarea'] = array(
+    '#type' => 'textarea',
+    '#title' => t('Changes Requested:'),
+    '#default_value' => '',
+    '#states' => array(
+      'invisible' => array(
+        ':input[name="inope_policy_review_status_radio"]' => array('value' => 0),
+      ),
+      'visible' => array(
+        ':input[name="inope_policy_review_status_radio"]' => array('value' => 1),
+      ),
+      'required' => array(
+        ':input[name="inope_policy_review_status_radio"]' => array('value' => 1),
+      ),
+    ),
+  );
+
+  $form['inope_policy_review_submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Submit'),
+  );
+  
+  $form['inope_policy_review_hidden_nid'] = array(
+    '#type' => 'hidden',
+    '#value' => $form_state['build_info']['args'][0]->nid,
```

Comment:
This is a bit opaque to the reader. I am sure you'd agree that $node->nid is a clearer statement of what we are setting the value to -- I think
$node = $form_state['build_info']['args'][0];
Would clear that up.

You can make it even more clear however by not using drupal get form as the callback and creating your own callback for the page -- which is what I was trying to get at. I think you thought I meant on the title callback, and its good you did what you did there -- to do that here you'd need a page callback where you pass the node object and then call drupal get form with an additional arg of the node.

This second half is like maybe not as important and I won't block functional on it -- but it's what I was expecting to see after my comments.